bundles/org.openhab.binding.mielecloud/src/main/resources/org/openhab/binding/mielecloud/internal/config/assets/css/main.css

```diff
@@ -0,0 +1,15023 @@
+/*! normalize.css v5.0.0 | MIT License | github.com/necolas/normalize.css */
```

Comment:
Is this really needed today?

Reply:
It is still needed to have a common initial state across all browsers.